### ISSUE TITLE
[Request] Equipment blacklist

### ISSUE THREAD

**Issue by Estrelaura** [qxeii/nox-renoxed#16]
I installed the mod on a server and had the shocking surprise of seeing zombies spawning with the admin ban hammer. A tag for blacklisting equipment would be welcome.

**Comment by qxeii** [comment]
I don't believe this mod affects entity equipment spawning. (Though correct me if I'm wrong).

If you did want to change this there are mods out there which do this, though it being added as a part of this mod is not out of the question.

I'm a bit busy right now so I don't have time to add it myself, but as always, anyone is free to make pull requests and I'll merge them assuming the changes are good.

**Comment by ihymyly** [comment]
When "Mob Gear Scales With Difficulty" is set to true, Nox does directly affect how mobs wear armor and equipment in general. Unfortunately, there aren't any mods that support configuration of natural mob equipment (at least none to my knowledge) as of 1.20.1, Fabric.

That said, its been almost a year since this request, and I too enjoy how Nox works in general. Any ability to configure or blacklist certain items would be greatly appreciated.

**Comment by ihymyly** [comment]
I may have found a mod that helps with this issue, but it is only available on Modrinth: [https://modrinth.com/mod/mob-management](https://modrinth.com/mod/mob-management)
